src/artifacts_poetry_plugin/ArtifactsDeployHelper.py

```diff
@@ -133,18 +133,40 @@ def get_poetry(self, root, files):
             poetry_root = Path(root)
             poetry = Factory().create_poetry(poetry_root)
             poetry.root_dir = poetry_root
-            logger.info(f"Found python project at {root}")
+            logger.debug(f"Found python project at {root}")
         return poetry
 
-    def log_non_deployable_packages(self, poetry_list):
-        package_set = set()
+    def get_package_dict(self, packages):
```

Comment:
Recommend using dictionary comprehension for this
i.e. 
package_dict = {x.name: x for x in packages}

Reply:
Fixed.